### PR TITLE
Add links colors

### PR DESCRIPTION
**Pull request by andersonba**
Class: `link`
Modifiers: `--green, --yellow, --orange, --red, --white, --gray, --black`

Extra: remove typo https://github.com/jusbrasil/monalisa/pull/25/files#diff-f33e5bb88b47daa6c3e68d0b1eda7d79L787

### CODE REVIEW

**Comment by andersonba** [comment]
@shankarcabus ? 5 days :(

File: dist/css/general.min.css [hunk 1]
```diff
@@ -645,6 +645,30 @@ html{font-family:Roboto,sans-serif;font-weight:400;font-size:62.5%;}
...
+.link--red:active{color:#922300;}
+.link--white{color:rgba(255,255,255,0.87);}
+.link--white:hover{color:#fff;}
+.link--white:active{color:#fff;}
+.link--gray{color:gray;}
+.link--gray:hover{color:#666;}
+.link--gray:active{color:#4d4d4d;}
+.link--black{color:#333;}
+.link--black:hover{color:#1a1a1a;}
+.link--black:active{color:#1a1a1a;}
```
Comment (rafaelverger) [inline dist/css/general.min.css]: I think that the default color should not be a modifier

Comment (andersonba) [inline dist/css/general.min.css]: The default color is blue, without modifier, just `link`, see: https://github.com/jusbrasil/monalisa/pull/25/files#diff-ca1cd9a9fdb371994559a0bc08840131R18

**Comment by andersonba** [comment]
@rafaelverger done

**Comment by rafaelverger** [comment]
LGTM! BTW nice job implementing `link-color` mixin

**Pull request merged**